### PR TITLE
issue-calendar-styles-fix-ng6

### PR DESCRIPTION
**Pull request by alfredo-dotcms**

### CODE REVIEW

File: src/styles/primeng-theme/components/_common.scss [hunk 1]
```diff
@@ -564,17 +564,23 @@ textarea,
...
-            text-align: center;
-            padding: 8px;
-            &.ui-state-highlight {
-                background-color: #d6d6d6;
-                color: #212121;
+        td {
+            a {
+                border: 0 none;
+                text-align: center;
+                padding: 8px;
```
Comment (fmontes) [inline src/styles/primeng-theme/components/_common.scss]: basic-padding?

File: src/styles/primeng-theme/components/_common.scss [hunk 2]
```diff
@@ -564,17 +564,23 @@ textarea,
...
+                &.ui-state-highlight {
+                    background-color: #d6d6d6;
```
Comment (fmontes) [inline src/styles/primeng-theme/components/_common.scss]: Where all this colors are comming from? no vars?

Comment (alfredo-dotcms) [reply src/styles/primeng-theme/components/_common.scss]: legacy colors, prob created before the existence of color variables

File: src/styles/primeng-theme/components/_calendar.scss [hunk 3]
```diff
@@ -4,6 +4,60 @@ p-calendar {
             @extend #inputtext-active-border;
         }
     }
+
+    /* DatePicker */
+    .ui-datepicker.ui-widget {
+        padding: 0;
+        .ui-datepicker-header {
+            -webkit-border-radius: 0px;
```
Comment (fmontes) [inline src/styles/primeng-theme/components/_calendar.scss]: We don't need prefixed anymore.

Also 0 is 0 it doesn't need unit.

File: src/styles/primeng-theme/components/_calendar.scss [hunk 4]
```diff
@@ -4,6 +4,60 @@ p-calendar {
...
+            -moz-border-radius: 0px;
+            border-radius: 0px;
+            border-top: 0 none;
+            border-left: 0 none;
+            border-right: 0 none;
```
Comment (fmontes) [inline src/styles/primeng-theme/components/_calendar.scss]: Do we need `0 none`?

**Pull request merged**